### PR TITLE
Add scheduling for the data validation metrics job

### PR DESCRIPTION
**Pull request by chelseatroy**
The following PR introduces a job for evaluating whether our recorded search terms are changing in ways that might invalidate assumptions on which we've built our sanitization model: https://github.com/mozilla/docker-etl/pull/73

THIS PR schedules THAT job. 

Things I need help on:
- Understanding whether this works and how to test it
- Whether I need to do anything to instantiate the `docker_image` located at the path mentioned in my `DAG`, and if so, what that is.

### CODE REVIEW

File: dags/search_term_data_validation.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+from airflow import DAG
+from utils.gcp import gke_command
+from airflow.sensors.external_task import ExternalTaskSensor
+from utils.constants import ALLOWED_STATES, FAILED_STATES
+from utils.tags import Tag
+
+default_args = {
+    "owner": "ctroy@mozilla.com",
+    "email": ["ctroy@mozilla.com"],
```
Comment (quiiver) [inline dags/search_term_data_validation.py:22]: I would like to have my email included in this please

**Comment by quiiver** [comment]
> Understanding whether this works and how to test it

I haven't tested this out yet but this mana page talks about deploying dags to a sandbox cluster for testing

https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=SRE&title=WTMO+Developer+Guide

**Comment by quiiver** [comment]
> Whether I need to do anything to instantiate the docker_image located at the path mentioned in my DAG, and if so, what that is.

I believe that after you merge the `docker-etl` PR the docker image will be built and pushed to google cloud registry. at that point it will be available to airflow

**Review by perrymcmanis144 — APPROVED** [review]
👍

Discussed with Chelsea, LGTM

**Pull request merged**